shared/src/persistence/dynamo/users/createMigratedPetitionerUser.js

```diff
@@ -36,6 +36,7 @@ exports.createMigratedPetitionerUser = async ({ applicationContext, user }) => {
     const response = await applicationContext
       .getCognito()
       .adminCreateUser({
+        MessageAction: 'SUPPRESS',
```

Comment:
don't want to send emails during migration for petitioners who haven't already been created